modules/ROOT/pages/Development/ExtractGameFiles.adoc

```diff
@@ -49,17 +43,35 @@ return to the directory selector and review the above steps.
 
 === Specify Custom Version and Mapping
 
-Before you can view any files you must specify the mapping information.
-Navigate to `Settings` > `General`.
+Unreal Engine 5 introduced changes in how assets are stored in pak files.
+You will need to provide FModel with extra information so that it can read the data correctly.
+
+==== The required files can be found in your game install directory at
+`/CommunityResources/FactoryGame.usmap`
+and `/CommunityResources/CustomVersions.json`. 
+
+To start navigate to `Settings` > `General`.
 In the "UE Versions" dropdown, select `GAME_UE5_2 (536870944)`,
 then press the `Custom Versions` button in the "Version Configuration" row.
 
+[WARNING]
```

Comment:
This warning is placed in a somewhat confusing order because it shows up before the file you're being asked to edit. I'll reword this section after it gets merged, just leaving a comment here as a note

Reply:
The idea was that people would see the emphasized bit about UE5 changes and file locations.
Based on this conversation: https://discord.com/channels/555424930502541343/601030071221878784/1190644182923088015

In regards to the warning at Line 57, they were introduced to the file names and locations at the beginning of this at Line 46. And the warning about Win10 vs Win11 should be just before they are editing the custom versions, no?